bin/MadGraph5_aMCatNLO/cards/production/13TeV/VV/WWTo4Q_5f_amcatnloFxFx/WWTo4Q_5f_amcatnloFxFx_madspin_card.dat

```diff
@@ -0,0 +1,15 @@
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#initialization parameters
+ set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+ set max_weight_ps_point 400  # number of PS to estimate the maximum for each event
```

Comment:
suggest to remove spaces in front of the 2 lines.

Reply:
done